### ISSUE TITLE
Support multiple upgrade schedules in a global infrastructure

### ISSUE THREAD

**Issue by klizhentas** [gravitational/teleport#30825]
A customer was planning to start leveraging the automatic agent update feature. The [maintenance schedule configuration](https://goteleport.com/docs/management/operations/self-hosted-automatic-agent-updates/?scope=enterprise#step-33-configure-the-maintenance-schedule) does not support timezones or multiple schedules.

 Teleport clusters include agents in several countries, but it looks like the CMC only supports a single global maintenance window for all agents.

There a should be a way to set separate utc_start_hour based on the agent label, or something similar.

 As-is, just based on a quick read-through, it looks like every agent in the cluster will disconnect itself during the same hour, worldwide.

**Comment by andrewbks** [comment]
Yeah more control over agent update schedules would be great, allowing for a staggered approach across different days etc..

**Comment by sclevine** [comment]
Addressed by [Managed Updates v2](https://goteleport.com/docs/upgrading/agent-managed-updates/).
